frontend/src/components/Connect/List/ActionsCell.tsx

```diff
@@ -59,7 +61,7 @@ const ActionsCell: React.FC<CellContext<FullConnectorInfo, unknown>> = ({
     stateMutation.mutateAsync(ConnectorAction.RESTART_FAILED_TASKS);
 
   return (
-    <Dropdown>
+    <Dropdown disabled={isReadOnly}>
```

Comment:
Not sure it's a good idea to apply this kind of rule for the whole component, as some elements in the dropdown might be allowed (like, r/o actions). There are also exclusions from r/o rule - see `SAFE_ENDPOINTS` in `io/kafbat/ui/config/ReadOnlyModeFilter.java:23`:
```
private static final Set<Pattern> SAFE_ENDPOINTS = Set.of(
      Pattern.compile("/api/clusters/[^/]+/topics/[^/]+/(smartfilters|analysis)$")
  );
```
So, actions related to topic analysis and smartfilters should be allowed despite being non-GET HTTP methods.

Reply:
Fixed. Applied additional isReadOnly check to only restart/remove connector actions.